docs/configuration/production.rst

```diff
@@ -205,7 +272,8 @@ Here's an example:
     [uwsgi]
     wsgi-file = app.wsgi
     enable-threads = true
-    http-socket = 127.0.0.1:8888
+    socket = /run/uwsgi/kinto.sock
```

Comment:
Both `/run` and `/var/run` are valid on debian. If `/var/run` is more cross-platform, let's go for it :)